java/com/globant/models/ShoppingCart.java

```diff
@@ -0,0 +1,111 @@
+package com.globant.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * This class models a shopping cart for a given user.

```

Comment:
There is no need for the "This class" part, just mentioning it's use works and makes it shorter